src/utils/common-utils.js

```diff
@@ -53,6 +53,27 @@ export function pathIsInSearch(searchVal, path) {
   return stringToSearch.includes(searchVal);
 }
 
+function processRequestBodyProperties(properties, stringToSearch) {
+  if (!properties) {
+    return '';
+  }
+
+  // We should use a separated string variable in order to exclude next error 'Uncaught RangeError: Invalid string length'
+  let recursiveStringToSearch = '';
+  stringToSearch = `${stringToSearch} ${Object.keys(properties || '').join(' ')}`;
+
+  Object.keys(properties).forEach((propertyName) => {
+    if (properties[propertyName].properties) {
+      recursiveStringToSearch = `${recursiveStringToSearch} ${processRequestBodyProperties(properties[propertyName].properties, stringToSearch)}`;
+    }
+    if (properties[propertyName].type === 'array') {
+      recursiveStringToSearch = `${recursiveStringToSearch} ${processRequestBodyProperties(properties[propertyName].items.properties, stringToSearch)}`;
```

Comment:
It seems like you are joining the string twice. the function should take an empty string initially and join it with `recursiveStringToSearch`


> I may be incorrect, just looking on a phone

Reply:
if (obj.properties[propertyName].properties) {
      recursiveresultStr = `${recursiveresultStr} ${joinObjectKeys(obj.properties[propertyName], resultStr)}`;
    }
if (obj.properties[propertyName].type === 'array') {
      recursiveresultStr = `${recursiveresultStr} ${joinObjectKeys(obj.properties[propertyName].items, resultStr)}`;
}

We can have both executing of these "if" statements in the context of the one function call. Therefore we have to join string from the first 'if' statement to the second one.